main.js

```diff
@@ -50,3 +50,76 @@ if (arrayOfTodos[i].completed === false) {
 }
 }
 
+// filtering branch
+const getUserID = () => {
+//clear the previous todos by removing all li as long as there is an ol
+const ol = document.querySelector('ol')
+if (ol) {
+  while (ol.firstChild) {
+    ol.removeChild(ol.firstChild)
+  }
+}
+// I had to use globalThis.filtered to make the "filtered" variable accesible global. Taking it out of the function broke the code for some reason
+globalThis.filtered = arrayOfTodos.filter(num => num.userId == document.getElementById('userId').value)
+
```

Comment:
not sure why. but i changed it to `let filtered = ...` and was able to access it just fine. i don't see you declaring the filtered variable anywhere else.

Reply:
Thanks so much for all of your feedback! It truly means a lot!